Example-Swift/ViewController.swift

```diff
@@ -13,25 +13,25 @@ import NYTPhotoViewer
 class ViewController: UIViewController, NYTPhotosViewControllerDelegate {
 
     @IBOutlet weak var imageButton : UIButton?
-    private let photos = PhotosProvider().photos
+    fileprivate let photos = PhotosProvider().photos
     
-    @IBAction func buttonTapped(sender: UIButton) {
+    @IBAction func buttonTapped(_ sender: UIButton) {
         let photosViewController = NYTPhotosViewController(photos: self.photos)
         photosViewController.delegate = self
-        presentViewController(photosViewController, animated: true, completion: nil)
+        present(photosViewController, animated: true, completion: nil)
         
         updateImagesOnPhotosViewController(photosViewController, afterDelayWithPhotos: photos)
     }
     
-    func updateImagesOnPhotosViewController(photosViewController: NYTPhotosViewController, afterDelayWithPhotos: [ExamplePhoto]) {
+    func updateImagesOnPhotosViewController(_ photosViewController: NYTPhotosViewController, afterDelayWithPhotos: [ExamplePhoto]) {
         
-        let delayTime = dispatch_time(DISPATCH_TIME_NOW, 5 * Int64(NSEC_PER_SEC))
+        let delayTime = DispatchTime.now() + Double(5 * Int64(NSEC_PER_SEC)) / Double(NSEC_PER_SEC)
```

Comment:
Random drive-by note:

In Swift 3, there's an overload of `+` that lets you add a `DispatchTime`  to `Double`s, so you could write this as `DispatchTime.now() + 5.0`.

Or, there's a new enum, `DispatchTimeInterval`, that has a `seconds(Int)` case, so you could also write this as `DispatchTime.now() + DispatchTimeInterval.seconds(5)`.